src/api/routes/projects.js

```diff
@@ -1,25 +1,37 @@
 const ProjectsService = require('../route-services/projects');
 
+const {
+  expressAuthorizationMiddleware,
+  expressAuthenticationOnlyMiddleware,
+} = require('../../utils/authMiddlewares');
+
 const projectsService = new ProjectsService();
 
 module.exports = {
-  'projects#update': (req, res, next) => {
-    projectsService.updateProject(req.params.projectUuid, req.body)
-      .then((data) => res.json(data))
-      .catch(next);
-  },
-  'projects#delete': (req, res, next) => {
-    projectsService.deleteProject(req.params.projectUuid)
-      .then((data) => res.json(data))
-      .catch(next);
-  },
-
-  'projects#get': (req, res, next) => {
-    projectsService.getProjects().then((response) => res.json(response)).catch(next);
-  },
-  'projects#getExperiments': (req, res, next) => {
-    projectsService.getExperiments(req.params.projectUuid)
-      .then((response) => res.json(response)).catch(next);
-  },
-
+  'projects#update': [
+    expressAuthenticationOnlyMiddleware,
```

Comment:
Wrong middleware?

Reply:
This has to be authOnly because we use `update` to create project and the accompanying experiment with the rbac_can_write that goes with it.